aws/b2b_personalization/modules/api_gateway/global/main.tf

```
@@ -0,0 +1,64 @@
+terraform {
+  required_version = ">= 1.0.0, < 2.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
```

let's use a fixed version to avoid surprises between version 
```suggestion
terraform {
  required_version = "1.3.3"

  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "4.37.0"
    }
  }
}
```